catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrMetacardClientImpl.java

```diff
@@ -164,16 +166,21 @@ public SourceResponse query(QueryRequest request) throws UnsupportedQueryExcepti
       query.setFacetMinCount(textFacetProp.getMinFacetCount());
     }
 
-    Serializable suggestQueryProp = request.getPropertyValue(SUGGESTION_QUERY_KEY);
-    Serializable suggestContextProp = request.getPropertyValue(SUGGESTION_CONTEXT_KEY);
-    Serializable suggestDictProp = request.getPropertyValue(SUGGESTION_DICT_KEY);
+    Serializable suggestQuery = request.getPropertyValue(SUGGESTION_QUERY_KEY);
+    Serializable suggestContext = request.getPropertyValue(SUGGESTION_CONTEXT_KEY);
+    Serializable suggestDict = request.getPropertyValue(SUGGESTION_DICT_KEY);
 
-    if (suggestQueryProp instanceof String && suggestContextProp instanceof String) {
+    if (suggestQuery instanceof String && suggestContext instanceof String) {
       query = new SolrQuery();
       query.setRequestHandler("/suggest");
-      query.setParam(SUGGEST_Q, (String) suggestQueryProp);
-      query.setParam(SUGGEST_CONTEXT_FILTER_QUERY, (String) suggestContextProp);
-      query.setParam(SUGGEST_DICT, (String) suggestDictProp);
+      query.setParam(SUGGEST_Q, (String) suggestQuery);
+      query.setParam(SUGGEST_CONTEXT_FILTER_QUERY, (String) suggestContext);
+      query.setParam(SUGGEST_DICT, (String) suggestDict);
```

Comment:
According to the discussion here https://github.com/codice/ddf/pull/2743#discussion_r162466371 it is okay for this to be null. However if it's non-null but not a String then this would fail. I don't see a problem adding another `instanceof` check.

Reply:
Up to you. 🙇